lib/adagrams.rb

```diff
@@ -0,0 +1,84 @@
+require "pry"
+SCORE_TO_LETTERS = {1 => ["a", "e", "i", "o", "u", "l", "n", "r", "s", "t"], 2 => ["d", "g"], 3 => ["b", "c", "m", "p"],
+  4 => ["f", "h", "v", "w", "y"], 5 => ["k"], 8 => ["j", "x"], 10 => ["q", "z"]}
+LETTERS_TO_QUANTITY = {"a" => 9, "b" => 2, "c" => 2, "d" => 4, "e" => 12, "f" => 2, "g" => 3, "h"=> 2, "i" => 9, "j" => 1, "k" => 1,
+    "l" => 4, "m" => 2, "n" => 6, "o" => 8, "p" => 2, "q"=> 1, "r" => 6, "s" => 4, "t" => 6, "u"=> 4, "v" => 2,
+    "w" => 2, "x" => 1, "y"=> 2, "z" => 1}
+def draw_letters
+  letter_bank = Array.new
+  10.times do
+    random_letter_picked = LETTERS_TO_QUANTITY.keys.sample
```

Comment:
So, you have a probability problem here. When you sample from the keys, the odds are not weighted by the number of each tile in the `letter_bank`.  Your results give a 1/26 chance to draw any letter, but that's not accurate is it?